File: app/src/main/java/com/liferay/wedeploy/util/LoadProfilePhoto.java
package com.liferay.wedeploy.util;

import com.bumptech.glide.Glide;
import de.hdodenhof.circleimageview.CircleImageView;

/**
 * @author Albertinin Santos
 */
public class LoadProfilePhoto {
	public static void load(CircleImageView photo) {

		Glide.with(photo.getContext())
			.load(PreferencesUtil.getStringByToken(photo.getContext(), KeysConstants.PHOTO_URL))
			.into(photo);
	}
}

File: app/src/main/java/com/liferay/wedeploy/util/LoadProfilePhoto.kt
package com.liferay.wedeploy.util

import com.bumptech.glide.Glide
import de.hdodenhof.circleimageview.CircleImageView


/**
 * @author Albertinin Santos
 */

class LoadProfilePhoto {
    companion object {

        fun load(photo: CircleImageView) {
            Glide.with(photo.context)
                .load(PreferencesUtil.getStringByToken(photo.context, KeysConstants.PHOTO_URL))
                .into(photo)
        }
    }
}